msal/application.py

```diff
@@ -673,6 +674,16 @@ def initiate_auth_code_flow(
             If the elapsed time is greater than this value,
             Microsoft identity platform will actively re-authenticate the End-User.
 
+        :param str response_mode:
+            OPTIONAL. Specifies the method with which response parameters should be returned.
+            For security purposes, the value should be form_post. In this mode, response parameters
+            will be encoded as HTML form values that are transmitted via the HTTP POST method and
+            encoded in the body using the application/x-www-form-urlencoded format.
+            Valid values can be either "form_post" for HTTP POST to callback URI or
+            "query" (the default) for HTTP GET with parameters encoded in query string.
+            More information on possible values
+            `here <https://openid.net/specs/oauth-v2-form-post-response-mode-1_0.html>`
+
```

Comment:
This entire paragraph is currently being inserted into the middle of other parameter's doc, though. Please move it into around line 690.